frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingslisteForTiltakstyper.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tiltaksgjennomforingrad } from "./Tiltaksgjennomforing";
+import { useTiltaksgjennomforinger } from "../../api/tiltaksgjennomforing/useTiltaksgjennomforinger";
+import { Alert, Heading, Loader } from "@navikt/ds-react";
+import { Tiltaksgjennomforing } from "mulighetsrommet-api-client";
+import styles from "./TiltaksgjennomforingslisteForTiltakstyper.module.scss";
+import tiltaksgjennomforingsStyles from "./Tiltaksgjennomforingeroversikt.module.scss";
+
+interface TiltaksgjennomforingerListeTiltakstypeProps {
+  tiltakstypeKode: string;
+}
+
+export function TiltaksgjennomforingslisteForTiltakstyper({
+  tiltakstypeKode,
+}: TiltaksgjennomforingerListeTiltakstypeProps) {
+  const { data, isLoading } = useTiltaksgjennomforinger();
```

Comment:
`useTiltaksgjennomforinger` vil vel bare returnere 50? Så jeg tror du må utvide api'et til å ta i mot en tiltakstypekode og få returnert alle tiltaksgjennomføringene for den koden. Slik det er nå så kan du i teorien få 50 tiltaksgjennomføringer, men ingen av de er knyttet til tiltakstypekoden du er interessert i og dermed vil ikke noe vises i frontend til brukeren